src/main/java/hng_java_boilerplate/user/dto/LogInDto.java

```diff
@@ -0,0 +1,31 @@
+package hng_java_boilerplate.user.dto;
+
+import jakarta.validation.constraints.NotBlank;
+
+public class LogInDto {
+    @NotBlank(message = "Email and password are required.")
+    private String email;
+    @NotBlank(message = "Email and password are required.")
+    private String password;
```

Comment:
Separate the error message. Email is required. Password is required

Reply:
You might as well use lomboks getters and setters

Reply:
for security reasons, being specific on what field is wrong, is that not bad?